Modules/MUON/MID/src/DigitsQcCheck.cxx

```diff
@@ -263,21 +263,56 @@ void DigitsQcCheck::beautify(std::shared_ptr<MonitorObject> mo, Quality checkRes
       }
       mHistoHelper.updateTitleWithNTF(histo);
       histo->SetStats(0);
-    } else {
+    } else if (mo->getName().find("BendHitsMap") != std::string::npos) { // Strips Display
+      // This matches both [N]BendHitsMap*
+      int maxStrip = 20; // 20kHz Max Display
+      auto histo = dynamic_cast<TH2F*>(mo->getObject());
+      mHistoHelper.normalizeHistoTokHz(histo);
+      histo->SetMaximum(zcontoursStrip.back());
+      histo->SetContour(zcontoursStrip.size(), zcontoursStrip.data());
+      histo->SetStats(0);
+    } else if (mo->getName() == "Hits") {
+      auto histo = dynamic_cast<TH1F*>(mo->getObject());
```

Comment:
same here

Reply:
Ok thanks for these changes and sorry for my late response.
When is it possible to have it at point2?

Reply:
Ok thanks again

Reply:
@ValerieRamillien are you planning to implement these proposed changes?

Reply:
Sorry for the delay. I just did it.